lib/engine/game/g_1822/step/buy_sell_par_shares.rb

```diff
@@ -33,9 +33,11 @@ def choices_ability(entity)
             return {} unless entity.company?
 
             choices = @game.company_choices(entity, :stock_round)
-            if !choices.empty? && entity.id == @game.class::COMPANY_OSTH
-              return {} if @bid_actions.positive? || @game.player_debt(entity.owner).positive?
+            if choices.any? && entity.id == @game.class::COMPANY_OSTH &&
```

Comment:
Did rubocop autocorrect do the `!empty?` -> `any?` thing? IIRC `!empty?` is more performant for the compiled JS.

Reply:
```suggestion
            if !choices.empty? && entity.id == @game.class::COMPANY_OSTH &&
```